java/main/com/google/privacy/differentialprivacy/SecureNoiseMath.java

```diff
@@ -85,4 +85,18 @@ public static double roundToMultipleOfPowerOfTwo(double x, double granularity) {
       return x;
     }
   }
+
+  /** Computes the closest double value that is larger than or equal to the provided long value. */
+  public static double nextLargerDouble(long n) {
+    double result = n;
+    long dif = (long) result - n;
+    return dif >= 0 ? result : Math.nextUp(result);
+  }
+
+  /** Computes the closest double value that is smaller than or equal to the provided long value. */
+  public static double nextSmallerDouble(long n) {
+    double result = n;
```

Comment:
can you please add an example in the comments? input and output and why this works?

Reply:
the same in the above method

Reply:
Added explanation to the documentation and an example in comments.